plugins/experimental/ssl_session_reuse/src/publish.cc

```diff
@@ -117,7 +117,11 @@ RedisPublisher::is_good()
 ::redisContext *
 RedisPublisher::setup_connection(const RedisEndpoint &re)
 {
-  auto my_id = (unsigned long long)::pthread_self();
+  uint64_t my_id = 0;
+  if (TSIsDebugTagSet(PLUGIN)) {
+    my_id = (uint64_t)pthread_self();
+  }
+
   TSDebug(PLUGIN, "RedisPublisher::setup_connection: Called by threadId: %llx", my_id);
```

Comment:
Shouldn't we put this line into the if block?

Reply:
You do it for some of other places. But not here?

Reply:
I can do it for this one easily.  However, there are more TSDebug() function calls in this function.

Reply:
OK.